server/routes/findInterventions/findInterventionsController.test.ts

```diff
@@ -49,4 +49,22 @@ describe(FindInterventionsController, () => {
         })
     })
   })
+
+  describe('GET /find-interventions/intervention/:id', () => {
+    it('responds with a 200', async () => {
+      const intervention = interventionFactory.build({
+        title: 'Better solutions (anger management)',
+        serviceCategory: { name: 'thinking and behaviour' },
```

Comment:
Do we need to specify a service category name if we're not making any assertions that depend on it?

Reply:
Another case where I'd like to blame it being a Friday - I meant to add an assertion for this... Have done so